test/libsolidity/syntaxTests/functionCalls/external_functions_with_calldata_args_assgined_to_function_pointers_of_memory_type.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    function f(function (string calldata) external) external {}
```

Comment:
`function f` here can be removed. Or maybe we could pass `ptr` to it inside of `main` so we don't get the unrelated (to this PR) unused variable warning.

Reply:
No need for passing the `ptr` to function `f`. The warning can be removed otherwise as well.

Pushed changes for that